podium-gateway/src/main/webapp/app/entities/organisation/organisation.model.ts

```diff
@@ -9,11 +9,28 @@
  */
 
 export class Organisation {
-    constructor(
-        public id?: number,
-        public uuid?: string,
-        public name?: string,
-        public shortName?: string,
-    ) { }
 
+    id?: number;
+    uuid?: string;
+    name?: string;
+    shortName?: string;
+    deleted?: boolean;
+    activated?: boolean;
+    organisationUuid?: string;
+
+    constructor(id?: number,
```

Comment:
We dont need all these values as constructor parameters as we never initialize a Request or Organisation like this (e.g. new Organisation(id, uuid, name, shortname, activated, deleted, organisationuuid);)